smarts/env/rllib_hiway_env.py

```diff
@@ -147,7 +149,13 @@ def step(self, agent_actions):
 
         for done in dones.values():
             self._dones_registered += 1 if done else 0
-        dones["__all__"] = self._dones_registered == len(self._agent_specs)
+        # TODO temporary solution for game of tag: stop the episode when 1 vehicle is done
+        # so that the other vehicle does not train when the opponent is not present, which
+        # causes noisy in training
+        dones["__all__"] = self._dones_registered > 0
```

Comment:
Will create a PR on adding configurable DoneCreteria for stopping simulation when certain vehicles are done, and wiill remove this if that PR can be merged